doc/mgr/dashboard.rst

```diff
@@ -280,11 +280,28 @@ Grafana on your preferred hosts, proceed with the following steps::
 
 After you have configured Grafana and Prometheus, you will need to tell the
 Ceph Manager Dashboard where it can access Grafana and what the credentials are
-to do so. This can be done by using the following commands::
+to do so, as well as the authentication method which shall be used.
```

Comment:
I wonder if this paragraph could be rephrased. How about the following:

"After you have set up Grafana and Prometheus, you will need to configure the
connection information that the Ceph Manager Dashboard will use to access Grafana.
This includes setting the authentication method to be used, the login credentials
as well as the URL at which the Grafana instance can be reached."

The URL and TCP port can be set by using the following command::

...